apps/src/gamelab/GameLab.js

```diff
@@ -43,6 +43,7 @@ import {
   postContainedLevelAttempt,
   runAfterPostContainedLevel
 } from '../containedLevels';
+import * as codeStudioLevels from '../code-studio/levels/codeStudioLevels';
```

Comment:
You could instead do:
`import { hasValidContainedLevelResults } from '../code-studio/levels/codeStudioLevels;'`
then elsewhere
`hasValidContainedLevelResult()` instead of `codeStudioLevels.hasValidContainedLevelResult()`